### PR TITLE
14.0 mig sale operating unit

### PR DESCRIPTION
**Pull request by cramosh**

### CODE REVIEW

**Comment by cramosh** [comment]
@AaronHForgeFlow Travis is indicating errors in the module tests that have nothing to do with the sale_operating_unit. How to proceed

**Comment by AaronHForgeFlow** [comment]
We need to migrate the branch 14.0 to use github actions instead of travis. That will be done in separate PR, I will check how to do that.

**Comment by cramosh** [comment]
> We need to migrate the branch 14.0 to use github actions instead of travis. That will be done in separate PR, I will check how to do that.

ok

**Comment by AaronHForgeFlow** [comment]
I mean to fix that here: https://github.com/OCA/operating-unit/pull/471 but  I am not sure if I did well, I will wait for others to check.

File: sale_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "version": "14.0.1.0.0",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+    "company",
+    "author": "ForgeFlow, "
+    "Serpent Consulting Services Pvt. Ltd.,"
+    "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/operating-unit",
+    "category": "Sales Management",
+    "depends": ["sale", "account_operating_unit", "sales_team_operating_unit"],
```
Comment (JuanyDForgeflow) [inline sale_operating_unit/__manifest__.py]: change dependecy "sale" to "sale_management" since it's not installing the module

Comment (JuanyDForgeflow) [reply sale_operating_unit/__manifest__.py]: Functional review installing the sales module manually is working ok

**Comment by francesco-ooops** [comment]
@cramosh @AaronHForgeFlow same issue as described [here](https://github.com/OCA/operating-unit/pull/464#issuecomment-1081883362)

**Comment by AaronHForgeFlow** [comment]
@cramosh I am sorry, but this PR should be closed in favor of https://github.com/OCA/operating-unit/pull/464, that was created before. Can you close it? You can reopen later on if needed.
@francesco-ooops  Thanks for pointing that out.

**Comment by francesco-ooops** [comment]
> @cramosh I am sorry, but this PR should be closed in favor of #464, that was created before. Can you close it? You can reopen later on if needed. @francesco-ooops Thanks for pointing that out.

I don't know if there are specific rules about this situation, but I think it's better to keep working on a PR where the dev is active rather than one that's been inactive for weeks, don't you think? 

It is important that everyone involved is aware that there are two PRs, but the community's interest is just reaching the fastest merge, regardless of who opened first

(also @cramosh 's original pr was older, see #365 )

**Comment by cramosh** [comment]
> > @cramosh I am sorry, but this PR should be closed in favor of #464, that was created before. Can you close it? You can reopen later on if needed. @francesco-ooops Thanks for pointing that out.
> 
> I don't know if there are specific rules about this situation, but I think it's better to keep working on a PR where the dev is active rather than one that's been inactive for weeks, don't you think?
> 
> It is important that everyone involved is aware that there are two PRs, but the community's interest is just reaching the fastest merge, regardless of who opened first
> 
> (also @cramosh 's original pr was older, see #365 )


Ok I will rectify the test and upload it again

**Comment by AaronHForgeFlow** [comment]
@francesco-ooops  it is ok to me to keep this open if @olveracuenca is not going to work on https://github.com/OCA/operating-unit/pull/464 anymore. I will ask in that PR.

**Comment by AaronHForgeFlow** [comment]
It fails 2 things now:

pre-commit (just run pre-commit run -a)

I think you have to create a new product in the test, instead of using a reference. That way you will be able to change the type

**Comment by AaronHForgeFlow** [comment]
Hi @cramosh , This PR is not abandoned https://github.com/OCA/operating-unit/pull/464 and it is green now. I think we should close this and continue on the other one. Sorry for that.

**Pull request closed without merging**